test/gov/setup_agbot.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+echo "setup_agbot.sh start"
+
+echo "Deleting old binaries..."
+rm $ANAX_SOURCE/anax
+rm $ANAX_SOURCE/cli/hzn
```

Comment:
Will this script only be run as part of the `test-remote-prebuilt` target? If so, the `clean` should have already removed these